runtimes/adapters/internal_agent.py

```
@@ -101,21 +107,70 @@ def required_dependencies(self) -> list[RuntimeDependency]:
+            # Prefer a lightweight endpoint; many Ollama installs respond on root
+            probe_url = f"{base}/v1/health" if base.endswith(":11434") else base
+            resp = httpx.get(probe_url, timeout=1.0)
+            if resp.status_code >= 200 and resp.status_code < 400:
+                return RuntimeHealth(
+                    runtime_id=self.RUNTIME_ID,
+                    available=True,
+                    details={"workspace_root": self._workspace_root, "provider": provider_label, "probe_url": probe_url},
+                )
+        except Exception:
+            # fall through to unavailable
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)